### DISCUSSION TITLE
How do to implement quality gates for GitHub Advanced Security at the PR level?

### DISCUSSION THREAD

**Discussion by andreissf** [github/codeql#15909] [General]
hi there,

anybody have any luck with implementing PR blockers if a high / critical vulnerability is found by GH Advanced Security in Azure DevOps?

Thanks.

**Comment by felickz** [comment]
Hi @andreissf, check out some of the open-source work that is happening over in this repo: https://github.com/microsoft/GHAzDO-Resources

- [pr-gating](https://github.com/microsoft/GHAzDO-Resources/tree/main/src/pr-gating#ghazdo-pr-gating) - has both a sample script and pipeline to show how to block/annotate on the PR in ADO.

**Reply by andreissf** [reply]
@felickz. I will take a look on that! Thanks a lot
